docs/faqs.rst

```diff
@@ -0,0 +1,87 @@
+  - You want to run Galaxy workflows from the command line.
+
+What is the difference between Galaxy and CWL? Which should I use?
+=============================
+
+Galaxy is a web-based scientific analysis platform which provides access to scientific software
+via a graphical interface. It is also a scientific workflow management system and allows chaining
+multiple individual tools together to form a complex pipeline, which can be executed as a single
+tool. If you place a high value on a graphical interface, usability and developing flexible, reusable
+tools which can easily be adapted for other purposes, Galaxy could be a good choice for you.
+
+CWL (Common Workflow Language) is a workflow specification, rather than a workflow management system.
+It aims to allow the description of scientific pipelines in a way that is independent of the particular
+workflow manager used; if this is important to you, using CWL would be an excellent idea. CWL workflows
+are supported by a variety of workflow managers, including Toil, cwltool, Arvados and Galaxy. 
+
+Both Galaxy and CWL place a high value on community, open-source code and FAIR data analysis.
+
+Can I develop Galaxy tools without Planemo?
+=============================
+
+Yes, you can, but we highly discourage you from doing so. Planemo encourages best practices in
+software development and follows guidelines agreed on by the Galaxy community, particularly
+test-driven development.
+
+How can I learn tool development?
+=============================
+
+We have a `tutorial`_ in this documentation and another incorporated into the
+Galaxy Training Network, which should get you off to a good start.
+
+How do I set up deployment to the ToolShed with Planemo?
+=============================
+
+We recommend using continuous integration (e.g. GitHub Actions) to achieve this and provide
+a `template`_ which you can use to set up a GitHub repository which automatically deploys tools
+to the ToolShed when a pull request is merged.
+
+If you don't want to handle tool deployment yourself, you can also submit your tool wrappers to
+a community repository such as the `IUC`_. This also has the advantage that your wrappers will be
+reviewed by experienced Galaxy users and developers.
+
+How can I automate workflow execution with Planemo?
+=============================
+
+Have a look at this `section`_ of the documentation, or this `training`_ provided by the Galaxy
+Training Network.
+
+How can I contribute to the project?
+=============================
+
+We would love to see new contributions to Planemo! Probably the easiest way to contribute is
+to this documentation; if you find any material which is unclear, out-of-date, or simply wrong,
+please `make a GitHub issue`_ or a pull request to our `GitHub repository`_. If you are comfortable
+writing code, have a look at our `list of issues`_ for ideas.
```

Comment:
Maybe simply refer to https://github.com/galaxyproject/planemo/blob/master/CONTRIBUTING.rst ?

Reply:
Yes, thank you!